cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/AbstractConnectOperator.java

```diff
@@ -707,11 +745,11 @@ protected boolean hasRestartAnnotation(CustomResource resource, String connector
     }
 
     /**
-     * Return the ID of the connector task to be restarted if the provided KafkaConnector resource instance has the strimzio.io/restart-task annotation
+     * Return the ID of the connector task to be restarted if the provided KafkaConnector resource instance has the strimzi.io/restart-task annotation
```

Comment:
These 2 changes are unrelated. Let me know if you prefer to fix them in a separate PR.